changelog/rwm-shared-error.dd

```diff
@@ -0,0 +1,3 @@
+The deprecation for read-modify-write operations on `shared` variables has ended
+
+Read-modify-write operations are not allowed for `shared` variables. Use $(REF core, atomic, atomicOp) instead.
```

Comment:
Please include an example that now fails to compile and show how to correct it.

Reply:
ok added